Saucy/indigo build error

I'm trying to build on saucy/indigo but getting the following error. Any ideas?

```
  File "/home/scpeters/ws/gazebo_ros_pkgs/src/gazebo_ros_pkgs/gazebo_plugins/cfg/GazeboRosCamera.cfg", line 6, in <module>
    from driver_base.msg import SensorLevels
ImportError: No module named msg
Built target nav_msgs_generate_messages_py
Built target nav_msgs_generate_messages_lisp
make[2]: *** [/home/scpeters/ws/gazebo_ros_pkgs/devel/include/gazebo_plugins/GazeboRosCameraConfig.h] Error 1
```

--- a/gazebo_plugins/CMakeLists.txt
+++ b/gazebo_plugins/CMakeLists.txt
@@ -120,6 +120,7 @@ catkin_package(
     gazebo 
     SDF
   )
+add_dependencies(${PROJECT_NAME}_gencfg ${catkin_EXPORTED_TARGETS})
 
 ## Executables
 add_executable(hokuyo_node src/hokuyo_node.cpp)

